thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DetectionJobResource.java

```diff
@@ -574,6 +576,8 @@ public Response generateAnomaliesInTrainingData(@PathParam("id") @NotNull long i
    * @param holidayEnds: holidayEnds in in ISO Format, ex: 2016-5-23T00:00:00Z,2016-6-23T00:00:00Z,...
    * @param features: a list of features separated by comma, ex: weight,score.
    *                Note that, the feature must be a existing field name in MergedAnomalyResult or a pre-set feature name
+   * @param pattern: user customization on pattern
```

Comment:
Please list the patterns here in the java doc.

Reply:
fixed.thanks.